src/components/Guilds/ActionsModal/DXDController.tsx

```diff
@@ -0,0 +1,67 @@
+import styled from 'styled-components';

```

Comment:
Obviously we dont yet have the configs ready to consume so I understand why its not built like that yet and is mostly hardcoded but we definitely shouldn't be baking in DXDController into actions modal components. This is a product that many teams externally to DXdao will use and so we need to make as much of it as possible configurable and avoid hardcoding like this. 
Instead this could be a Contract component representing one contract and all the calls that can be made to it, we could then pass it as harcoded props for now.

Reply:
Good Point, I'll make this more flexible.